pycalphad/tests/test_equilibrium.py

```diff
@@ -190,17 +197,23 @@ def test_eq_missing_component():
                     {v.T: 1523, v.X('AL'): 0.88811111111111107,
                      v.X('CO'): 0.11188888888888888, v.P: 101325})
 
+@pytest.mark.solver
 def test_eq_ternary_edge_case_mass():
     """
     Equilibrium along an edge of composition space will still balance mass.
     """
     eq = equilibrium(ALCOCRNI_DBF, ['AL', 'CO', 'CR', 'VA'], ['L12_FCC', 'BCC_B2', 'LIQUID'],
-                     {v.T: 1523, v.X('AL'): 0.88811111111111107,
-                      v.X('CO'): 0.11188888888888888, v.P: 101325}, verbose=True)
+                     {v.T: 1523, v.X('AL'): 0.8881111111,
+                      v.X('CO'): 0.1118888888, v.P: 101325}, verbose=True)
     mass_error = np.nansum(np.squeeze(eq.NP * eq.X), axis=-2) - \
-                 [0.88811111111111107, 0.11188888888888888, 0]
-    assert np.all(np.abs(mass_error) < 0.01)
-
+                 [0.8881111111, 0.1118888888, 1e-10]
+    assert_allclose(eq.GM.values, -97913.542)  # from Thermo-Calc 2017b
+    result_chempots = eq.MU.values.flatten()
+    assert_allclose(result_chempots[:2], [-86994.575, -184582.17], atol=0.1)  # from Thermo-Calc 2017b
+    assert result_chempots[2] < -300000  # Estimated
```

Comment:
It's complicated to determine what the chemical potential should be, because it's unclear what the limit of numerical precision in the composition conditions in TC/OC is. This test may need to be rethought, but I'd like to defer that to another issue.